vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionSpaceController.java

```diff
@@ -49,12 +51,27 @@
     @Autowired
     private SequenceHistory sequenceHistory;
 
-    @RequestMapping(value = "/exhibit/space/{id}")
-    public String space(@PathVariable("id") String id, Model model) {
+    @RequestMapping(value = { "/exhibit/space/{id}", "/preview/{previewId}/space/{id}", "/preview/{previewId}" })
```

Comment:
`/preview/{previewId}` should be handled in the home controller (as it is the equivalent to `/`)